interfaces/apparmor/backend.go

```diff
@@ -458,14 +458,9 @@ func downgradeConfinement() bool {
 			return false
 		}
 	case release.DistroLike("arch"):
-		if strings.HasSuffix(kver, "-hardened") {
-			if cmp, _ := strutil.VersionCompare(kver, "4.17.4"); cmp >= 0 {
-				// The linux-hardened 4.17.4+ package has
-				// apparmor enabled, do not downgrade the
-				// confinement template.
-				return false
-			}
-		}
+		// The default kernel has AppArmor enabled since 4.18.8, the
```

Comment:
I assume the checks are no longer needed because the new dependency on arch ensures that the right kernel gets pulled in, correct?

Reply:
Arch is on the latest mainline or LTS kernel. The mainline is 4.18 at the moment, so it's fresh enough to have all the features we need (IIRC we are fine AppArmor wise with anything 4.14+). The LTS kernel (4.14.72) does not have AppArmor enabled so the backend is effectively unused. On top of this people are expected to update whole distro when installing stuff because there's no way to ensure rpm or deb like binary compatibility so one is always running the latest and greatest (for better or worse).